db/initial_inserts.sql

```diff
@@ -0,0 +1,53 @@
+--Initial inserts, ad inserts here that should ALWAYS be in the database
+INSERT INTO courses
+    (course_code,title, department)
+VALUES
+    ('ECSE 428', 'Software engineering in practice', 'ECSE');
+
+INSERT INTO tags
+    (name)
+VALUES
+    ('Engineering');
+
+INSERT INTO course_tag
+    (course_code, tag_name)
+VALUES
+    ('ECSE 428', 'Engineering');
+
+INSERT INTO `
+users`
```

Comment:
```suggestion
users`
(username, email, password)
```
Shouldn't you specify columns here in case of further modifications to db? Or do we assume that whoever changes db will also update this?